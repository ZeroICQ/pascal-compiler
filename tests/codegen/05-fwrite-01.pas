 begin
    fwrite(20);
    fwrite(1.2);
    fwrite(123.3e-12);
 end.