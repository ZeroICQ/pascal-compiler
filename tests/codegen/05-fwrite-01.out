201.21.233e-010